### ISSUE TITLE
Constructed and Limited progress not updating

### ISSUE THREAD

**Issue by mahamoti64** [Razviar/mtgap#292]
https://mtgarena.pro/progress/?Mahamoti64#tab-Constructed
https://mtgarena.pro/progress/?Mahamoti64#tab-Limited

These pages show no new data since October 31 / November 1.
However, my matches are still being logged.  i.e. 

https://mtgarena.pro/progress/?Mahamoti64#tab-Matches
and
https://mtgarena.pro/matches/?Mahamoti64

I did have a stretch of a few weeks where the tracker wasn't running, but it's been running now for the last few weeks (as indicated by the match data).

Can the constructed and limited progress pages start updating again?

I'm running Windows 10, standalone app v2.2.14

Thanks,
Jeff

**Comment by first-stanza** [comment]
I'm having the same issue. Haven't updated since dec 9. Though i'm not sure if I have played limited since then but constructed every day.

Also now a days updating in profile daily progress is really slow and many times not showing all the gains when it decides to update. 

Tried to uninstall and install again. Tried the account wipe. Tried resyncing the logs. nothing helps. I really like the plugin but it's pretty broken on my end at the moment.

https://mtgarena.pro/progress/?gfhih#tab-Constructed
